contentcuration/contentcuration/assets/channelList/utils.js

```diff
@@ -0,0 +1,39 @@
+import omit from 'lodash/omit';
+import cloneDeep from 'lodash/cloneDeep';
+import isEqual from 'lodash/isEqual';
+import store from './store';
+
+const DIRTY_KEY = '_lastSaved';
```

Comment:
Usually I think of "dirty" as a boolean, but here it's being used to describe the complete prior state, which isn't really the same thing.

Maybe just call it like it is:

```js
const LAST_SAVED_KEY = '_lastSaved';
```

Reply:
Yeah - that is definitely more descriptive - I guess I was just looking at the state of my apartment and thinking about dirt.